applications/frontier/src/frontier_sup.erl

```diff
@@ -0,0 +1,58 @@
+%%%-------------------------------------------------------------------
+%%% @copyright (C) 2010-2013, 2600Hz
+%%% @doc
+%%%
+%%% @end
+%%% @contributors
+%%%   SIPLABS, LLC (Maksim Krzhemenevskiy)
+%%%-------------------------------------------------------------------
+-module(frontier_sup).
+
+-behaviour(supervisor).
+
+-export([start_link/0]).
+-export([init/1]).
+
+-include("frontier.hrl").
+
+%% Helper macro for declaring children of supervisor
+-define(CHILDREN, [?WORKER('frontier_ensure_rates_in_syscfg')
```

Comment:
As I said in the module, I'd have a generic `frontier_init` module that spawns a process and returns 'ignore' instead.